daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/UDFunctions.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.dpath
+
+case class UDFunctionCall(recipes: List[CompiledDPath], functionClassObject: Object)
+  extends FNArgsList(recipes) {
+
+  lazy val maybeEvaluate = functionClassObject.getClass.getMethods.collect { case p if p.getName == "evaluate" => p }
```

Comment:
This "evaluate" string has appeared in at least one other place. Should we make it a variable somewhere so avoid this magic string?